bin/postinstall.sh

```diff
@@ -3,10 +3,12 @@ id -u iip &> /dev/null || useradd iip
 
 # create directory for logging
 mkdir -p /var/log/iip
-
-# give permission to iip user
 chown -R iip:iip /var/log/iip
 
+# create directory for work dir
+mkdir -p /data
+chown -R iip:iip /data
```

Comment:
Ownership of this directory is root, and should be configured by puppet.  Directories below that /data/staging/atforwarder, /data/staging/dbb, /data/staging/oods need to be owned by the user transferring  the files to file store (in this case /data/staging/atforwarder) which should also be set up via puppet.

Reply:
Usually the root file system is pretty small. Something like this might be better in /var/tmp/data, since /var/tmp survives reboots. And we can have a bigger space in /var/tmp.